### DISCUSSION TITLE
MatSee Plus Switch Tuya 16A not supported

### DISCUSSION THREAD

**Discussion by mpmguedes** [Koenkk/zigbee2mqtt#14322] [Devices]
Hi, 
Hello
I bought a matsee plus switch that has good specs (16A) and that promised to be able to integrate by zigbee2mqtt.
https://es.aliexpress.com/item/1005004128359494.html?spm=a2g0o.store_pc_groupList.8148356.16.52787c769Ov48O&pdp_npi
In conclusion, the switch despite appearing reviews on the company's shopping page saying that they were able to integrate, mine simply appears as unsupported.
After searching the company's equipment in zigbee2mqtt's list of supported equipment it doesn't really show up.

I would like some help even if it is for manual integration.

Thanks

**Comment by Koenkk** [comment]
Please fill in https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=new+device+support&template=new_device_support.yaml&title=%5BNew+device+support%5D%3A+
